main.go

```diff
@@ -59,6 +59,8 @@ type Config struct {
 	Draft         string          `env:"draft,opt[yes,no]"`
 	PreRelease    string          `env:"pre_release,opt[yes,no]"`
 	FilesToUpload string          `env:"files_to_upload"`
+	APIUrl        string          `env:"api_base_url"`
```

Comment:
The linter did not fail for the `APIUrl` ?
I think it should be `APIURL` ( uhh... it looks terrible 🙃  )

Reply:
Fixed.